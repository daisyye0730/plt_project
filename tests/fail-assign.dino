int main() {
    string word;
    word = 5;
}