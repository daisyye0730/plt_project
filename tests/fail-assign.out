Fatal error: exception Failure("illegal assignment string = int in word = 5")
